EU4toV2/Source/Configuration.cpp

```diff
@@ -105,6 +105,10 @@ void Configuration::instantiate(std::istream& theStream, bool (*doesFolderExist)
 		commonItems::singleString debugString(theStream);
 		debug = (debugString.getString() == "yes");
 	});
+	registerKeyword(std::regex("randomiseRgos"), [this](const std::string& unused, std::istream& theStream){
+		commonItems::singleString debugString(theStream);
```

Comment:
Nit: usually these are given a meaningful name to match what they hope to extract, for instance ```randomiseRgosString```

Reply:
Done.